.editorconfig

```diff
@@ -0,0 +1,19 @@
+# top-most EditorConfig file
```

Comment:
Can you remove this file so the PR is just changes related to autotools->cmake?

Reply:
Yea, we can add it in another pull request.